### PR TITLE
Club: implement button styles for interactive states in the read more block button

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club: implement button styles for interactive states in the read more block button

image

### CODE REVIEW

File: club/style.css [hunk 1]
```diff
@@ -63,15 +68,7 @@ a {
...
-}
-
-:is(
-	.wp-block-search__button,
-	.wp-block-file .wp-block-file__button,
-	.wp-block-button__link,
-	.wp-elements-button,
+	.wp-block-read-more,
 ):where(:active) {
 	border-style: dotted;
```
Comment (mikachan) [inline club/style.css:73]: Only a small detail, but I think this should be `dashed` rather than `dotted`, when looking at the comps.

**Review by mikachan — COMMENTED** [review]
This is looking great, I've just left one other comment.

File: club/style.css (hunk 1 above)
Comment (matiasbenedetto) [reply club/style.css:73]: good catch! Change ready!

**Review by mikachan — APPROVED** [review]
Looks good to me!

**Pull request merged**